README.md

```diff
@@ -43,25 +46,27 @@ module.exports = {
 ```JSON
 {
   "chunk-vendors": {
-    "js": "/js/chunk-vendors.fc40696c.js",
-    "jsMap": "/js/chunk-vendors.fc40696c.js.map"
+    "js": ["/js/chunk-vendors.fc40696c.js"],
+    "js.map": ["/js/chunk-vendors.fc40696c.js.map"]
   },
   "app": {
-    "css": "/css/app.eb829ccc.css",
-    "js": "/js/app.dd31cdcb.js",
-    "jsMap": "/js/app.dd31cdcb.js.map"
+    "css": ["/css/app.eb829ccc.css"],
+    "js": ["/js/app.dd31cdcb.js"],
+    "js.map": ["/js/app.dd31cdcb.js.map"]
   }
 }
 ```
 
 ## Options
 
-There are 2 input options available 
-
 | Option | Description |
 | ------------- |-------------|
-| outputDir | Folder in which to output the JSON file. If the folder does not exist, we'll try to create it |
-| filename | Name of the outputed JSON file |
+| excludeFile | `RegExp` or `function(filename, chunk definition object) => bool`. Exclude HMR chunks by default (file names ending with `.hot-update.js`). |
```

Comment:
There is to much stuff in description. With this changes, we should create two new columns here. Please add one with `Defaults` and one with `Type`. This is where we would respectively put defaults and types, not mention then in description as this representation is much harder to consume.